### PR TITLE
Stripe app name and icon

### PR DESCRIPTION
**Pull request by lemieux**
Here you go, this makes use of the new endpoint and display the name and icon of the Stripe app.

@alavers @dannytranlx @Mario54

### CODE REVIEW

File: src/js/smooch.jsx [hunk 1]
```diff
@@ -146,11 +147,17 @@ export class Smooch extends Observable {
         }).then((loginResponse) => {
             store.dispatch(setUser(loginResponse.appUser));
             store.dispatch(updateReadTimestamp(getReadTimestamp()));
-            
+
             if (loginResponse.publicKeys) {
                 store.dispatch(setPublicKeys(loginResponse.publicKeys));
-            }
 
+                if (loginResponse.publicKeys.stripe) {
```
Comment (Mario54) [inline src/js/smooch.jsx]: We should probably find a better way to indicate that stripe is configured for the app in the `init` response

**Comment by Mario54** [comment]
:+1:

File: src/js/smooch.jsx (hunk 1 above)
Comment (lemieux) [inline src/js/smooch.jsx]: My guess is that it's the best way... don't have the public key? no stripe for you

Comment (mspensieri) [inline src/js/smooch.jsx]: The alternative would be to have an extra key in the response, `app.stripeEnabled = !!publicKeys.stripe`, which formalizes the assumption that public key means stripe enabled.

Comment (lemieux) [inline src/js/smooch.jsx]: yeah we could do that too

Comment (lemieux) [inline src/js/smooch.jsx]: since we already have `gcmEnabled`, that could be similar

**Comment by lemieux** [comment]
ehhh I added the spinner back in this PR by error.... just review too plez :) @Mario54

File: src/js/components/action.jsx [hunk 2]
```diff
@@ -57,7 +76,10 @@ export class ActionComponent extends Component {
                                     stripeKey={ publicKeys.stripe }
                                     email={ user.email }
                                     amount={ this.props.amount }
-                                    currency={ this.props.currency.toUpperCase() }>
+                                    currency={ this.props.currency.toUpperCase() }
+                                    name={ stripeAccount.appName }
+                                    image={ stripeAccount.iconUrl }
+                                    closed={ this.onStripeClose.bind(this) }>
```
Comment (alavers) [inline src/js/components/action.jsx]: Is there any preference for doing this vs `this.onStripeClose = this.onStripeClose.bind(this)` in the constructor?

File: src/js/smooch.jsx (hunk 1 above)
Comment (Mario54) [inline src/js/smooch.jsx]: Yeah having a `stripeEnabled` key sounds better

File: src/js/components/action.jsx (hunk 2 above)
Comment (lemieux) [inline src/js/components/action.jsx]: in this case the constructor is already define but I usually prefer this to not have to add a constructor

**Comment by alavers** [comment]
:+1:

**Comment by Mario54** [comment]
:+1:

File: src/js/smooch.jsx (hunk 1 above)
Comment (lemieux) [inline src/js/smooch.jsx]: I'll leave it like that for now and add a card about adding that to the backend

**Pull request merged**